docs/sdks/sdk-ref-python.md

```diff
@@ -236,6 +238,30 @@ If you're using server-side integration (see [Publisher Integration Guide, Serve
         unmapped_identity = unmapped_identities.get("email1@example.com")
         reason = unmapped_identity.get_reason()
    ```
+### Monitor rotated salt buckets
```

Comment:
Add an intro line below the heading and before the procedure:
To monitor salt buckets, follow these steps: